src/components/Jobs/components/Job/jobPropTypes.js

```diff
@@ -0,0 +1,27 @@
+import PropTypes from 'prop-types';
+
+const datePropType = PropTypes.oneOfType([
+  PropTypes.string,
```

Comment:
Please change `datePropType` to be just string. In mocks convert dates to string for this. Eventually, we are expecting dates to be strings.